src/amo/components/UserProfile/index.js

```diff
@@ -206,28 +201,18 @@ export class UserProfileBase extends React.Component<InternalProps> {
       >
         <ul>
           {reviews.map((review) => {
-            const reviewBodySanitized = sanitizeHTML(nl2br(review.body), [
-              'br',
-            ]);
+            const byLine = (
+              <Link
+                title={i18n.gettext('Browse the reviews for this add-on')}
+                to={`/addon/${review.addonSlug}/reviews/`}
+              >
+                {i18n.moment(review.created).fromNow()}
```

Comment:
> Also, it is surprising to me that each review links to the browse reviews page (will users be able to figure this out?).

That's a pretty common pattern (it would be better to directly link to the review though).

> Anyway, I'm just thinking out loud. I don't have a concrete change request here but it feels like some part of the abstraction is missing.

I did not find a better way to fix that, but I can think about it again.

> As an aside, is the UX spec for user profile reviews out of date? It doesn't match what we have.

mmh yes, I think this has changed a bit but we also did not implement the latest UI for the add-on card/list (this is a global change).